StatisticsTests/Distributions/ParametricBootstrap.cs

```diff
@@ -37,7 +38,8 @@ public void BootstrapWorksForLP3_Test(double mean, double standardDeviation, dou
                 randomNumberArray[i] = random.NextDouble();
             }
             IDistribution testBootstrapDistribution = distributionLP3.Sample(randomNumberArray);
-            double actual = testBootstrapDistribution.Mean;
+            Statistics.Distributions.LogPearson3 tbd = (Statistics.Distributions.LogPearson3)testBootstrapDistribution;
+            double actual = tbd.Mean;
```

Comment:
is there something more description we could use than tbd? tbd looks like to be decided

Reply:
yeah i actually kinda liked tbd because of that fact. it would be testbootstrapdistributionLP3